src/main/java/handler/OperationHandlerBalance.java

```diff
@@ -0,0 +1,17 @@
+package handler;
+
+import dao.StorageDao;
+import model.FruitTransaction;
+
+public class OperationHandlerBalance implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
the same for other handlers